perfmetrics/scripts/ls_metrics/run_ls_benchmark.sh

```diff
@@ -6,6 +6,6 @@ echo Installing pip and fuse..
 sudo apt-get install fuse -y
 sudo apt-get install pip -y
 echo Installing requirements..
-pip install -r perfmetrics/scripts/ls_metrics/requirements.txt --user
+pip install -r requirements.txt --user
 echo Running script..
-python3 perfmetrics/scripts/ls_metrics/listing_benchmark.py perfmetrics/scripts/ls_metrics/config.json --command "ls -R" --num_samples 30 --upload --message "Testing CT setup."
+python3 listing_benchmark.py ls_metrics/config.json --command "ls -R" --num_samples 30 --upload --message "Testing CT setup."
```

Comment:
shouldn't  ls_metrics/config.json be just config.json since we are already inside ls_metrics directory?

Reply:
fixed